### PR TITLE
Gilded Darknight

### LINKED ISSUE #366: Add tracking for Gilded Darknight
https://www.wowhead.com/item=186549/gilded-darknight
Drop: Adamant Vaults
Zone: Torghast, Tower of the Damned

### PR DESCRIPTION
**Pull request by godejord**
Added another easy one. [Gilded Darknight](https://www.wowhead.com/item=186549/gilded-darknight), a pet from Adamant Vaults in Torghast. This closes #366 :)

### CODE REVIEW

**Comment by Duckwhale** [comment]
Thanks! Only one thing, should we not reuse the existing localization phrase from the other item?

> This item can only drop in the Adamant Vaults section of Torghast

It's not quite as specific, but it's one less phrase to translate.

File: Locales.lua [hunk 1]
```diff
@@ -1897,6 +1897,8 @@ L["The Mawshrooms are obtained from treasure nodes called Invasive Mawshroom in
 L["Darkmaul is obtained by feeding a friendly NPC in Korthia called Darkmaul 10 Tasty Mawshroom"] = true
 L["Adamant Vaults Cell"] = true
 L["This item can only drop in the Adamant Vaults section of Torghast."] = true
+L["Gilded Darknight"] = true
+L["This pet can only drop in the Adamant Vaults section of Torghast."] = true
```
Comment (Duckwhale) [inline Locales.lua:1901]: This one seems a bit redundant (see other comment).

**Comment by godejord** [comment]
> Thanks! Only one thing, should we not reuse the existing localization phrase from the other item?
> 
> > This item can only drop in the Adamant Vaults section of Torghast
> 
> It's not quite as specific, but it's one less phrase to translate.

I just thought it sounded a bit wrong calling a pet for an item, but I'm good either way. Updated!

**Comment by Duckwhale** [comment]
Thank you! It probably doesn't really matter, but I think most people will understand "item" to refer to the pet cage (AKA "item that allows you to learn the pet") and not the pet itself (as summoned from the pet journal).

**Comment by Duckwhale** [comment]
Also I'm squashing this, as the second commit is really just a fixup :)

**Pull request merged**